docs(react-router): Add docs for distributed tracing (#13467)
getsentry/sentry-docs: docs/platforms/javascript/guides/react-router/index.mdx

});
```

Update your `entry.server.tsx` file:

```tsx {diff} {filename: entry.server.tsx}
+import * as Sentry from '@sentry/react-router';
 import { createReadableStreamFromReadable } from '@react-router/node';
 import { renderToPipeableStream } from 'react-dom/server';
 import { ServerRouter } from 'react-router';
 import { type HandleErrorFunction } from 'react-router';

+const handleRequest = Sentry.createSentryHandleRequest({
+  ServerRouter,
+  renderToPipeableStream,
+  createReadableStreamFromReadable,
+});

export default handleRequest;

export const handleError: HandleErrorFunction = (error, { request }) => {
  // React Router may abort some interrupted requests, don't log those
  if (!request.signal.aborted) {
+   Sentry.captureException(error);
    // optionally log the error so you can see it
    console.error(error);
  }
};

// ... rest of your server entry
```

> <Expandable title="Do you need to customize your handleRequest function?">
>   If you need to update the logic of your `handleRequest` function you'll need to include the provided Sentry helper functions (`getMetaTagTransformer` and `wrapSentryHandleRequest`) manually:

Reviewer: Do I understand this correctly that this more manual approach is necessary when users have their own `handleRequest` function already? If so, I think we should point this out more clearly because I didn't fully understand the "If you need to update the logic of your `handleRequest` function" part. (Maybe that's just me though, so feel free to disregard)

```suggestion
<Expandable title="Do you have your own handleRequest function?">
  If already defined your own `handleRequest` function you'll need to include the provided Sentry helper functions (`getMetaTagTransformer` and `wrapSentryHandleRequest`) manually:
```

(feel free to go with different wording as you see fit

Author: yeah that is correct, BUT whenever a user creates this hook, the handleRequest function is already defined – so I only want to encourage this path when they need som special logic in their function .. not sure how to name this though

Reviewer: Maybe I don't fully understand this yet but I guess what you mean is that when you run `react-router reveal` there already is a predefined `handleRequest` and this is the one users _don't_ need to add Sentry stuff to? 
So it's just for the case that users have a custom behaviour in the function already prior to adding Sentry? 

I don't think this is a blocker btw, so if you wanna merge this today, feel free. Maybe we have a minute tomorrow to chat about it this quickly in person
